app/controllers/questions/backtaxes_controller.rb

```diff
@@ -2,12 +2,28 @@ module Questions
   class BacktaxesController < QuestionsController
     include AnonymousIntakeConcern
     skip_before_action :require_intake
+    before_action :load_possible_filing_years, only: [:edit, :update]
     layout "intake"
 
     def current_intake
       Intake::GyrIntake.new
     end
 
+    def edit
+      # skip this question if they answered that they only haven't filed for the current year
+      # and save on the intake that they need help for the current year
+      if current_triage.present? && only_current_tax_year_not_filed(current_triage)
```

Comment:
i looked at this a couple of times and kept thinking of alternate ways to approach it, and each time ended up coming back to understanding why you selected this approach.